pkg/render/intrusion_detection.go

```diff
@@ -61,10 +61,17 @@ const (
 	IntrusionDetectionControllerPolicyName = networkpolicy.TigeraComponentPolicyPrefix + IntrusionDetectionControllerName
 	IntrusionDetectionInstallerPolicyName  = networkpolicy.TigeraComponentPolicyPrefix + "intrusion-detection-elastic"
 
+	ADAPIObjectName          = "anomaly-detection-api"
+	ADAPIObjectPortName      = "anomaly-detection-api-https"
+	ADAPITLSSecretName       = "anomaly-detection-api-tls"
+	ADAPIExpectedServiceName = "anomaly-detection-api.tigera-intrusion-detection.svc"
+	ADAPIPolicyName          = networkpolicy.TigeraComponentPolicyPrefix + ADAPIObjectName
+	adAPIPort                = 8080
+
 	ADPersistentVolumeClaimName            = "tigera-anomaly-detection"
 	DefaultAnomalyDetectionPVRequestSizeGi = "10Gi"
 	adAPIStorageVolumeName                 = "volume-storage"
-	adAPIVolumePath                        = "/storage"
+	adAPIStoragePath                       = "/var/" + ADAPIObjectName + "/storage"
```

Comment:
It is more common to use `/media` to mount storage type devices as documented in https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/media.html. How difficult is it to change this to `/media/ADAPIObjectName/storage`?

Reply:
udpated to using `/mnt/...` https://github.com/tigera/operator/pull/2300/commits/ea759041f6c39bb7b8c7c7eec2a90a7e0cc69b7b